config/install/migrate_plus.migration.custom_peertube_migration_tn.yml

```diff
@@ -1,8 +1,8 @@
 langcode: en
 status: true
 dependencies: { }
-id: custom_peertube_migration
-label: 'Extract peertube media to file'
+id: custom_peertube_migration_tn
```

Comment:
Naming everything here as a "custom" migration isn't necessary and doesn't add any meaningful information about the module.  This module could simply be "peertube_migration" and these migration ids "peertube_migration_tn", etc.